Environment_oM/Elements/BoundaryZone.cs

```diff
@@ -0,0 +1,50 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using System.Collections.Generic;
+using BH.oM.Geometry;
+using BH.oM.Base;
+using BH.oM.Analytical.Elements;
+using BH.oM.Environment.Fragments;
+using System.ComponentModel;
+using BH.oM.Dimensional;
+
+namespace BH.oM.Environment.Elements
+{    
+    [Description("A zone of the external faces of an Environment Mesh.")]
+    public class BoundaryZone : BHoMObject, IEnvironmentObject
+    {
+        /***************************************************/
+        /**** Properties                                ****/
+        /***************************************************/
+        
+        [Description("The indecies of the faces in the Mesh3D which this BoundaryZone is refering to.")]
+        public virtual List<int> FaceIndecies { get; set; } = new List<int>();
```

Comment:
```suggestion
        public virtual List<int> FaceIndices { get; set; } = new List<int>();
```
This will need tracking though in any Engine methods. Not sure there are any yet - in https://github.com/BHoM/BHoM_Engine/pull/1884/files or https://github.com/BHoM/Rhinoceros_Toolkit/pull/167/files @kThorsager to double check 😄

Reply:
Not to worry, no Engine methods were created for the Environment objects, I'll leave that to the Environment people